NAMESPACE

```diff
@@ -52,6 +52,7 @@ export(object_name_linter)
 export(object_usage_linter)
 export(open_curly_linter)
 export(paren_brace_linter)
+export(pipe_call_linter)
```

Comment:
my bad :sweat_smile:

Reply:
I tend to forget `devtools::document()` as well 😆